src/applications/vaos/services/mocks/index.js

```diff
@@ -223,6 +223,24 @@ export default [
       };
     },
   },
+  {
+    path: /vaos\/v0\/facilities\/limits/,
+    response: (url, { _groups }) => {
+      const data = [];
+      if (url.includes('983'))
```

Comment:
I think I would add another condition in here that pushes an item if the url includes 984, with `numberOfRequests` set to 1. That matches the logic of the old mock and ensures that there's not an empty array returned, which I don't believe we'd see in production.